docs/source/cleanvision/imagelab.rst

```diff
@@ -0,0 +1,9 @@
+Imagelab
+========
+
+.. automodule:: src.cleanvision.imagelab
```

Comment:
If you've made the `sys.path` change, this now can be just `cleanvision.imagelab` and similarly in other `.rst` files